test/utils/describeConformanceV5.js

```diff
@@ -201,6 +205,38 @@ function testThemeStyleOverrides(element, getOptions) {
         ).to.toHaveComputedStyle(testStyle);
       }
     });
+
+    it("respect theme's styleOverrides specific slot", function test() {
```

Comment:
add new test for specific slot. @mnajdova should it be another PR or this PR is fine?

Reply:
Let's handle this in a separate PR. We can just disabl this test for the component with Todo and handle it in a follow up. I wouldn't add this logic in the `describeConformance` as it is not a conformant thing :)